### PR TITLE
INFRA: Initialize Build Project

### PR DESCRIPTION
**Pull request by sdozier99**
#4

### CODE REVIEW

File: InternTrack.Core.Api.Infrastructure.Build/Program.cs [hunk 1]
```diff
@@ -0,0 +1,70 @@
+﻿using ADotNet.Clients;
```
Comment (hassanhabib) [inline InternTrack.Core.Api.Infrastructure.Build/Program.cs:1]: We nee copyrights at the top - always at every `.cs` file

File: InternTrack.Core.Api.Infrastructure.Build/Program.cs [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+               },
+                 new RestoreTask
+                 {
+                     Name = "Restoring Nuget Packages"
+                 },
+
+                 new DotNetBuildTask
+                 {
+                     Name = "Building Project"
```
Comment (hassanhabib) [inline InternTrack.Core.Api.Infrastructure.Build/Program.cs:53]: indentations here aren't done right. This seems like a copied code. Do it line by line by hand.

File: InternTrack.Core.Api.Infrastructure.Build/Program.cs [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+                 },
+
+                 new TestTask
+                 {
+
```
Comment (hassanhabib) [inline InternTrack.Core.Api.Infrastructure.Build/Program.cs:58]: Why do we have an empty task here? does it have a title?

File: InternTrack.Core.Api.Infrastructure.Build/InternTrack.Core.Api.Infrastructure.Build.csproj [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net7.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>disable</Nullable>
```
Comment (hassanhabib) [inline InternTrack.Core.Api.Infrastructure.Build/InternTrack.Core.Api.Infrastructure.Build.csproj:7]: Both of these need to be disabled

File: InternTrack.Core.Api.Infrastructure.Build/Program.cs [hunk 5]
```diff
@@ -0,0 +1,70 @@
+﻿using ADotNet.Clients;
+using ADotNet.Models.Pipelines.GithubPipelines.DotNets;
+using ADotNet.Models.Pipelines.GithubPipelines.DotNets.Tasks;
+using ADotNet.Models.Pipelines.GithubPipelines.DotNets.Tasks.SetupDotNetTaskV1s;
+
+var githubPipeline = new GithubPipeline
```
Comment (hassanhabib) [inline InternTrack.Core.Api.Infrastructure.Build/Program.cs:6]: Where's the namespace? scope? classname?

Comment (Pwongchaiya) [reply InternTrack.Core.Api.Infrastructure.Build/Program.cs:6]: @hassanhabib In the Sofee video for this section you did'nt put in a namespace, scope, or classname in program.cs. Do you want me to show him how to do it?

**Comment by glhays** [comment]
> #4

You need to put Closes before the issue number. Requires a space between for FYI.
If you type Closes then space, then the hash tag you will see a list of open issues.

**Comment by sdozier99** [comment]
Thank you.

File: InternTrack.Core.Api.Infrastructure.Build/Program.cs (hunk 5 above)
Comment (hassanhabib) [reply InternTrack.Core.Api.Infrastructure.Build/Program.cs]: @Pwongchaiya yup show him.

**Pull request closed without merging**